modules/core/src/main/java/org/locationtech/jts/io/OrdinateFormat.java

```diff
@@ -0,0 +1,64 @@
+/*
+ * Copyright (c) 2019 Martin Davis.
+ *
+ * All rights reserved. This program and the accompanying materials
+ * are made available under the terms of the Eclipse Public License v1.0
+ * and Eclipse Distribution License v. 1.0 which accompanies this distribution.
+ * The Eclipse Public License is available at http://www.eclipse.org/legal/epl-v10.html
+ * and the Eclipse Distribution License is available at
+ *
+ * http://www.eclipse.org/org/documents/edl-v10.php.
+ */
+
+package org.locationtech.jts.io;
+
+import java.text.DecimalFormat;
+import java.text.DecimalFormatSymbols;
+
+/**
+ * Formats numeric values for ordinates
+ * in a consistent, accurate way.
+ * <p>
+ * The format has the following characteristics:
+ * <ul>
+ * <li>It is consistent in all locales (in particular, the decimal separator is always a period)
+ * <li>Scientific notation is avoided 
+ * <li>The maximum number of decimal places reflects the available precision
+ * </ul> 
+ * 
+ * @author mdavis
+ *
+ */
+public class OrdinateFormat
+{
+  /**
+   * The number of fraction digits to accommodate a reasonable ordinat value
+   */
+  public static final int MAX_FRACTION_DIGITS = 18;
+  
+  public static OrdinateFormat DEFAULT = new OrdinateFormat();
+
+  private DecimalFormat format;
+  
+  public static DecimalFormat createFormat(int maximumFractionDigits) {
```

Comment:
private?

Reply:
currently has to be `public` because it's used by `WKTWriter`.  But your suggested change below would let this become `private`.